### ISSUE TITLE
How can I get the link to an attached image?

### ISSUE THREAD

**Issue by TheAttilaTheFun** [Rapptz/discord.py#1667]
I can use `message.attachments` to get a list of attachments but I want the link to the image and the link only. Is there an easy way to do this? Or would I be better off trimming the contents of the list? If you could ELI5 I'd appreciate it, if not I'll find a way to understand :)

**Comment by Vexs** [comment]
I'm not quite sure I understand your issue here.
`message.attachments` is a list of dicts on the async branch(0.16.x), and attachment objects on the rewrite branch(1.0.0a). It's as simple as fetching the relevant key/attribute. 

Do you perhaps mean the embedded message that discord adds for when someone posts a link?

As a further note, github issues aren't really the place to ask for programming help. Our [support discord](https://discord.gg/r3sSKJJ) is more suited for that. Issues are more for issues with the lib.

**Comment by TheAttilaTheFun** [comment]
I apologize, I read on the docs to come over here to ask questions an saw some posted in the past. I knew this wasn't the "best" place to ask questions, but I didn't know that there was a specific discord.py discord. I'll head over there, Thanks!
